data_structure/segment_trees.cpp

```diff
@@ -0,0 +1,121 @@
+/**
```

Comment:
please change the structure to this:
```cpp
/**
  * @file
  * @brief Implementation of [Segment Trees](https://en.wikipedia.org/wiki/Segment_tree)
  * to find maximum number in an array and its index
  * in logarithmic time - \f$O(N * \log(N))\f$.
  * @author [Lasha Bukhnikashvili](author's GitHub profile link here)
  */
```

Reply:
add your GitHub profile link where it says : * author's GitHub profile link here*